python/dicom_summary.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+import traceback
+
+import lib.dicom.summary_make
+import lib.dicom.summary_write
+import lib.exitcode
+
+parser = argparse.ArgumentParser(description=(
+        'Read a DICOM directory and print the DICOM summary of this directory '
+        'in the the console.'
+    ))
+
+parser.add_argument(
+    'directory',
```

Comment:
```suggestion
    '--directory',
```

However, it might be best to use the lorisgetop library we already have instead of adding a new dependency

Reply:
Not using `--` means that it is a positional argument instead of a named one. I would think this is better here `dicom_archive.py <source_dir> <target_dir>` ?